### PR TITLE
pd documentation

### PR DESCRIPTION
**Pull request by tohsin**
# Improved Documentation for PD controller in the tutorial

Further break down of what a PD controller was, was given, as well as explanations of the choice of equation we choose to implement.

### CODE REVIEW

File: docs/tutorials/dataset_creation/point_maze_dataset.py [hunk 1]
```diff
@@ -160,18 +159,79 @@ def _check_valid_cell(self, cell):
...
+#
+# Now back to our controller as stated previously, for the D4RL task we use a PD controller and we
+# follow the same theme as what we have stated before as can be seen below. The :math:`Error` is equlivalent
+# to the difference between the :math:`\text{goal}_\text{pose}` and :math:`\text{agent}_\text{pose}` and we replace the derivative term :math:`(d(\text{Error}) / dt)` with
+# the velocity of the the agent :math:`v_{\text{agent}}`, we can think of this as a measure of the speed at which the agent
+# is approaching the target position. When the agent is moving quickly towards the target, the
+# derivative term will be larger, contributing to a stronger corrective action from the controller.
+# On the other hand, if the agent is already close to the target and moving slowly, the derivative term will be smaller,
+# resulting in a less aggressive control action.
 # references.
```
Comment (balisujohn) [inline docs/tutorials/dataset_creation/point_maze_dataset.py:228]: These lines are probably a typo.

**Review by balisujohn — COMMENTED** [review]
Lines which say "references." maybe included by accident. Otherwise, ready for merge :^)

File: docs/tutorials/dataset_creation/point_maze_dataset.py (hunk 1 above)
Comment (tohsin) [reply docs/tutorials/dataset_creation/point_maze_dataset.py:228]: Oh I suspect I was trying to justify why we had a pure velocity term as opposed to a first order differential error rate. 
 Yeah i think the reference can go

**Comment by balisujohn** [comment]
LGTM :^)

**Review by balisujohn — APPROVED** [review]

**Pull request merged**